src/this.js

```diff
@@ -19,6 +27,9 @@ const checkPassword = function comparePasswords(passwordToCompare) {
   // use `this` to access the object's `password` property.
   // do not modify this function's parameters
   // note that we use the `function` keyword and not `=>`
+  if (passwordToCompare == this.password) {
```

Comment:
Same thing: `return passwordToCompare === this.password;`.  Make sure to use `===` over `==`